### PR TITLE
update header

### PR DESCRIPTION
**Pull request by martinmcneela**
BEFORE CREATING THIS PR, have you bumped the package JSON where appropriate? Yup

Ticket / Why this is needed:
There is way to much duplicated markup in our apps, I am exporting an app header (the base compound components are still available) that takes a few props which will result in reducing the code in each of our application and changes in many places when they need to be made.

This is the first PR I will open another one to tidy up the header a little more once this has flown into the apps.

You can use this branch to see how the update will work in app: https://github.com/edozo/edozo-occupier-plans-fe/tree/update-header

### CODE REVIEW

File: src/EdozoLogo/assets/maps-logo.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...98Zm-5.236 6.446c-.572-.594-.858-1.342-.858-2.244 0-.902.286-1.65.858-2.222.572-.594 1.298-.88 2.178-.88.88 0 1.606.286 2.178.88.594.572.88 1.32.88 2.222 0 .902-.286 1.65-.88 2.244-.572.572-1.298.858-2.178.858-.88 0-1.606-.286-2.178-.858Zm17.049-8.052c-1.54 0-2.728.528-3.564 1.606V15h-2.838v15.4h2.838v-5.698c.836 1.078 2.024 1.606 3.564 1.606 1.474 0 2.75-.55 3.784-1.672 1.056-1.122 1.584-2.508 1.584-4.136 0-1.628-.528-2.992-1.584-4.114-1.034-1.122-2.31-1.694-3.784-1.694Zm-2.706 8.052c-.572-.594-.858-1.342-.858-2.244 0-.902.286-1.65.858-2.222.572-.594 1.298-.88 2.178-.88.88 0 1.606.286 2.178.88.594.572.88 1.32.88 2.222 0 .902-.286 1.65-.88 2.244-.572.572-1.298.858-2.178.858-.88 0-1.606-.286-2.178-.858Zm12.457-4.62c0-.55.55-.902 1.276-.902.814 0 1.408.374 1.76 1.122l2.42-1.32c-.858-1.496-2.398-2.332-4.18-2.332-1.188 0-2.178.308-2.992.946-.792.616-1.188 1.474-1.188 2.552 0 2.288 2.046 2.97 3.674 3.388 1.122.286 2.002.616 2.002 1.21 0 .638-.506.946-1.518.946-1.122 0-1.826-.484-2.156-1.43l-2.464 1.386c.814 1.738 2.354 2.618 4.62 2.618 1.254 0 2.31-.308 3.146-.902.858-.616 1.276-1.496 1.276-2.618.022-2.354-2.068-3.124-3.674-3.454-1.122-.286-2.002-.616-2.002-1.21Z" fill="#00D4FF"/></svg>
```
Comment (iza-pik) [inline src/EdozoLogo/assets/maps-logo.svg:1]: We are changing width only for maps and insight logo here, but not occupiers, which makes it not correctly aligned in the product switch.

File: src/Header/AppHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  stageUrl: string;
+}
+
+const defaultProducts: ProductInfo[] = [
+  {
+    appName: '',
+    description: '',
+    productionUrl: 'https://dashboard.edozo.com/',
+    stageUrl: 'https://dashboard.edozo.co/',
+  },
```
Comment (iza-pik) [inline src/Header/AppHeader.tsx:30]: Just noticed that in the header story, this adds an extra list item in the product switch.

**Review by iza-pik — COMMENTED** [review]
This looks great 👍  Just a couple of things I noticed I think could be adjusted a bit.

File: src/Header/AppHeader.tsx (hunk 2 above)
Comment (martinmcneela) [reply src/Header/AppHeader.tsx:30]: ah yeah, i'll remove that one - I had is to we had that option in the story controls

**Review by iza-pik — APPROVED** [review]

**Pull request merged**